src/microbots/utils/path.py

```diff
@@ -1,7 +1,51 @@
 import os
+from dataclasses import dataclass
 from pathlib import Path
 
 
+@dataclass
+class FolderMountInfo:
+    path_valid: bool
+    base_name: str
+    abs_path: str
+
+
+def get_file_mount_info(file_to_mount: str) -> FolderMountInfo:
```

Comment:
If you're planning to abstract it out of the Bot, I would suggest to have a single abstraction for both folders and files. Also, keep the src_path, dst_path, type (a folder or a file), permission (rw, ro), mount-type (mount or file-copy) under that abstraction. So, you can handle all the cases in a single abstraction. And that can be maintained at `util/env_mount.py` instead of `util/path.py`

Reply:
As suggested removed `get_folder_mount_info` and `get_file_mount_info` to single generic function called 

`get_path_info` and replaced all the files correctly

<img width="927" height="261" alt="image" src="https://github.com/user-attachments/assets/5d5db59a-3903-4550-8a07-eda451773e1c" />
`

creating `util/env_mount.py` may not be required for now or I am getting why do we need create that correctly.